lib/commands/file-movement.js

```diff
@@ -15,32 +14,11 @@ const CONTAINER_PATH_PATTERN = new RegExp(`^${CONTAINER_PATH_MARKER}([^/]+)/(.*)
 const CONTAINER_TYPE_SEPARATOR = ':';
 const IFUSE_CONTAINER_DOCUMENTS = 'documents';
 const CONTAINER_DOCUMENTS_PATH = 'Documents';
+const IO_TIMEOUT = 30000;
```

Comment:
I'd say the timeout to 5 minutes or even more. Folders can be quite large in size

Reply:
@mykola-mokhnach 30 seconds should be fine. It is also crazy that we map the folders into a buffer in node. How much memory do we consume then?

Reply:
at least the same amount that the content of zipped folder would consume. Although, I don't have any better solution for that since selenium API does not implement streamed downloads. We must keep everything in memory, convert into base64-string and then put into the response body

Reply:
with maximum speed of USB2.0 connection of about 40MB/s we could download 1.2 GB within 30 seconds. Appium's limit of request body size is 1GB. Yeah, this should suffice :)